invokeai/app/api/routers/board_images.py

```diff
@@ -53,3 +67,66 @@ async def remove_board_image(
     except Exception as e:
         raise HTTPException(status_code=500, detail="Failed to update board")
 
+
+@board_images_router.post(
+    "/batch",
+    operation_id="create_board_images_from_list",
+    responses={
+        201: {"description": "Board images were created successfully"},
+    },
+    status_code=201,
+    response_model=CreateBoardImagesFromListResult,
+)
+async def create_board_images_from_list(
+    board_images: list[BoardImage] = Body(
+        description="The board images to create", embed=True
```

Comment:
What is the use case for having a list of BoardImages vs a list of Images with one board_id? Are there front-end actions that can trigger this for multiple board/image combinations?

Reply:
Yes, a batch may have images from any board, and this allows it to support multi select in an intuitive way. Imagine you want to make a board out of your current batch

Reply:
I decided to just make this a list of image names with a single `board_id`, we can handle the other behavior if the actual need arises